protocol.py

```diff
@@ -121,9 +121,13 @@ def handleEvent_withReplyEvent_(self, event, replyEvent) -> None:  # noqa: N802
           and not is_wine
           and not config.auth_force_localserver
       )):
+    # This could be false if you use auth_force_edmc_protocol, but then you get to keep the pieces
+    assert sys.platform == 'win32'
     # spell-checker: words HBRUSH HICON WPARAM wstring WNDCLASS HMENU HGLOBAL
     from ctypes import windll  # type: ignore
-    from ctypes import POINTER, WINFUNCTYPE, Structure, byref, c_long, c_void_p, create_unicode_buffer, wstring_at
+    from ctypes import (  # type: ignore
```

Comment:
Why *this* `# type: ignore` ?

Reply:
because it seems that mypy ignores the assert above for the imports, and thus decides that those imports dont exist (WINFUNCTYPE for the second one, windll for the first)

Reply:
Ah, fair enough.